commons/jdk11/src/main/java/org/infinispan/commons/jdkspecific/ProcessInfo.java

```diff
@@ -24,6 +24,10 @@ public static ProcessInfo getInstance() {
       return new ProcessInfo(ProcessHandle.current());
    }
 
+   public static Process of(java.lang.Process process) {
+      return new Process(process.toHandle());
```

Comment:
Failure from CI:

`17:24:33  13:22:27.631 [ERROR] /home/infinispan/workspace/Infinispan_PR-8297/commons/jdk11/src/main/java/org/infinispan/commons/jdkspecific/ProcessInfo.java:[28,13] error: Process is abstract; cannot be instantiated`

Reply:
Perhaps this was meant to be:

```
-   public static Process of(java.lang.Process process) {
-      return new Process(process.toHandle());
+   public static ProcessInfo of(Process process) {
+      return new ProcessInfo(process.toHandle());
```

Reply:
@tristantarrant can you please take a look?